tests/std/tests/GH_001914_Cached_position/test.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <ranges>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+template <class View>
+class test_cache : public ranges::_Cached_position<View, test_cache<View>> {
+public:
+    constexpr test_cache(View view) noexcept : _view(move(view)) {}

```

Comment:
This unconditional `noexcept` is no bueno. (Also on 14.) Since they're not significant to the test, I'll just remove them.